src/packages/client/src/runtime/getPrismaClient.ts

```diff
@@ -43,11 +43,32 @@ import { printStack } from './utils/printStack'
 import { serializeRawParameters } from './utils/serializeRawParameters'
 import { validatePrismaClientOptions } from './utils/validatePrismaClientOptions'
 const debug = Debug('prisma-client')
+const ALTER_RE = /^alter/i
```

Comment:
I think we could improve this and ignore any white space before "alter" with a regex like `/^(\s*)alter/i`

Because currently if there is one space or line return before "ALTER" the regex will not match it.

Reply:
Great point

Reply:
Changed it to `/^(\s*alter\s)/i`